playbooks/roles/edxapp/tasks/customer_theme.yml

```diff
@@ -25,7 +29,7 @@
     repo: "{{ item.value.customer_theme_source_repo }}"
     version: "{{ item.value.customer_theme_version }}"
     accept_hostkey: yes
-    key_file: "/edx/app/edxapp/.ssh/id_rsa"
+    key_file: "{{ edxapp_git_identity }}"
```

Comment:
this fix is likely to be not needed, but using the variable which is on `/edx/app/edxapp/edxapp-git-identity`.